great_expectations/expectations/core/expect_column_values_to_be_in_set.py

```diff
@@ -56,17 +56,17 @@ class ExpectColumnValuesToBeInSet(ColumnMapExpectation):
                 "my_col",
                 [2,3]
             )
-        {
+        \{
             "success": false
-            "result": {
+            "result": \{
```

Comment:
Seems to resolve the matter?

Reply:
Not saying we definitely shouldn't do this, but:
* This is totally not ergonomic
* Does this work with the gallery build?

Reply:
I'm not sure to be perfectly honest. I don't like the solution I feel like there's got to be a better way here?